### ISSUE TITLE
Daily Tournament quests

### ISSUE THREAD

**Issue by vasius** [dalaranwow/dalaran-wow#5305]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
everyday the quests: A Blade Fit For A Champion -or- The Edge Of Winter -or- A Worthy Weapon are  not available... making 2 seals less per day
**Expected behaviour**: Tell us what should happen instead
One of the listed quests should be available
**Steps to reproduce the problem**:

1. Log in 
2. Go to questgivers
3. Cry cause no quest to be taken ( -2 from the 5 total seals u can have in a day )

**Include proofs for this behaviour**
http://wowwiki.wikia.com/wiki/Quest:A_Blade_Fit_For_A_Champion_(Alliance_Valiant)
You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
